dev-libs/libpqmarble/libpqmarble-2.0.0_p20250109.ebuild

```
@@ -0,0 +1,53 @@
+# Copyright 2025 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v3 or later
+
+EAPI=8
+
+COMMIT="f240b2ec7d5cdacb8fdcc553703420dc5101ffdb"
+
+inherit flag-o-matic meson vala
+
+DESCRIPTION="Utility library for GNOME apps"
+HOMEPAGE="https://gitlab.gnome.org/raggesilver/marble"
+SRC_URI="https://gitlab.gnome.org/raggesilver/marble/-/archive/${COMMIT}/marble-${COMMIT}.tar.bz2"
+
+S="${WORKDIR}/marble-${COMMIT}"
+
+LICENSE="GPL-3"
+SLOT="0/2"
+KEYWORDS="~amd64"
+
+DEPEND="
+	>=gui-libs/gtk-4.0:4
+"
+RDEPEND="${DEPEND}"
+
+BDEPEND="
+	>=dev-lang/vala-0.56
+	dev-libs/gobject-introspection
+	virtual/pkgconfig
+"
+
+# Fix CSS provider API compatibility across Vala versions
+# https://gitlab.gnome.org/raggesilver/marble/-/issues/12
+PATCHES=(
+	"${FILESDIR}/${PN}-2.0.0-css-provider.patch"
+)
+
+src_prepare() {
+	default
+	vala_setup
+}
+
+src_configure() {
+	# Workaround for Vala-generated C code pointer type warnings
+	# See: https://gitlab.gnome.org/GNOME/vala/-/issues/1408
+	#      https://bugs.gentoo.org/892708
+	append-cflags -Wno-incompatible-pointer-types
+
+	meson_src_configure
+}
```

```suggestion
src_configure() {
	meson_src_configure
}
```

vala_setup 中已经添加过 -Wno-incompatible-pointer-types ，所以不需要再 append 一次

---

https://github.com/gentoo/gentoo/blob/734379cd03859634c0ac6e29392850c7dce1cdeb/eclass/vala.eclass#L177